Ecommerce/ecommerce_app/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+
+class Product(models.Model):
+    product_name = models.CharField(max_length = 255)
```

Comment:
```suggestion
    name = models.CharField(max_length=255)
```

no need to add product prefix as model name is product as well